app.test.js

```diff
@@ -19,4 +19,84 @@ describe('Server', () => {
     });
   });
 
+  describe('GET /api/v1/projects/:id', () => {
+    it('should return a happy status code of 200 and a single project object if the project exists', async () => {
+      const expectedProject = await database('projects').first();
+        delete expectedProject.created_at;
+        delete expectedProject.updated_at;
```

Comment:
What's the thought behind deleting these? Is this because we are only returning the rest of the object in the response?

Reply:
I used the delete code here because the values for the created_at & updated_at properties were causing issues with testing. The values for these properties were mismatched with regard to type (string vs integer) in the data being delivered from Supertest versus the object grabbed from our Database.

An alternative to deleting the properties would be the approach below (credit to Allie). Stringifying then parsing the SuperTest response fixes the mismatch with the DB format.

const cleanedProject = JSON.parse(JSON.stringify(expectedProject))